Sources/swift-build/loosef.swift

```diff
@@ -0,0 +1,14 @@
+import PackageType
```

Comment:
What is `loosef`, loose functions? Can this be something more readable like helpers or util etc.?